lib/fizzy/execute.cpp

```diff
@@ -414,6 +419,71 @@ T fnearest(T value) noexcept
         return t;
 }
 
+
+union Num32
+{
+    unsigned int i32;
+    float f32;
+};
+
+union Num64
+{
+    uint64_t i64;
+    double d64;
+};
+
+float fabs(float input) noexcept
+{
+    Num32 result;
```

Comment:
`fizzy::bit_cast` must be used instead of type punning via `union Num32`.

Reply:
Hi i am really not sure so i must ask, do you think something like that 
float fabs(float input) noexcept
{
    auto result = bit_cast<unsigned int>(input);
    result &= ABS_MASK;
    return bit_cast<float>(result);
} 
Thanks for your answer.